src/main/kotlin/day11/FileExam.kt

```
@@ -0,0 +1,23 @@
+    /// writeText의 경우 덥어 씌워짐
+    file.writeText("Hello, World!\n")
+    /// appendText의 경우 이어 씌워짐
+    file.appendText("Hello, World!2")
+
+    println(file.readText())
+
+    try {
+        val file2 = File("test2.txt")
+        /// Exception in thread "main" java.io.FileNotFoundException: test2.txt (No such file or directory)
+        println(file2.readText())
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)